sig-scheduling/annual-report-2021.md

```diff
@@ -23,45 +24,48 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 -->
 
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - [902 - NonPreempting PriorityClass](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/902-non-preempting-priorityclass) - 1.24 
+     - [1258 - Default Pod Topology Spread](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1258-default-pod-topology-spread) - 1.24
+     - [1923 - Prefer Nominated Node](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1923-prefer-nominated-node) - 1.24
+     - [2249 - Namespace Selector for Pod Affinity](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2249-pod-affinity-namespace-selector) - 1.24
```

Comment:
these are great, but 1.24 is this year, so save this for the 2022 report (or tweak them to the stability phase they met in 1.23 in 2021) :)

Reply:
updated :)